src/extensions/project-extensions.ts

```diff
@@ -106,7 +106,7 @@ module.exports = (toolbox: GluegunToolbox): void => {
             QUESTIONS.template,
         ]);
 
-        const command = `${NPM7_PREFIX} && npx ionic start ${name} blank`;
+        const command = `${NPM7_PREFIX} && npx @ionic/cli start ${name} blank`;
```

Comment:
When the name of the package and the command are different, you need to use the -p syntax:

```suggestion
        const command = `${NPM7_PREFIX}&& npx -p @ionic/cli ionic start ${name} blank`;
```